src/connectors/_internal/hyperdrive.ts

```diff
@@ -0,0 +1,20 @@
+import type { Hyperdrive } from "@cloudflare/workers-types/experimental";
+
+async function getCloudflareEnv() {
+  try {
+    // @ts-ignore - cloudflare:workers is only available in CF runtime
+    const cfEnv = await import("cloudflare:workers");
+    return cfEnv.env;
+  } catch {
+    return {};
+  }
+}
+
+export const getHyperdrive = async (bindingName: string) => {
+  const env = await getCloudflareEnv();
+  const binding: Hyperdrive = env[bindingName] || globalThis.__env__?.[bindingName];
```

Comment:
We shall migrate to new `env` (from `cloudflare:env` [docs](https://developers.cloudflare.com/workers/runtime-apis/bindings/#how-to-access-env)) this is only  supported in legacy nitro/unenv setups

Reply:
https://github.com/nitrojs/nitro-cloudflare-dev/issues/58